integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1980,4 +2162,98 @@ public static ExecResult kubectlpatch(String domainUid, String domainNS, String
     return exec(cmd, true);
   }
 
+  /**
+   * Run script to delete DomainHome dir.
+   *
+   * @throws Exception exception
+   */
+  public static void deleteDomainHomeDir(String scriptDir, String namespace, String domainUid) throws Exception {
+
+    String jobName = "create-weblogic-sample-domain-job";
+    StringBuffer cmdDelJob = new StringBuffer("kubectl get job ");
+    cmdDelJob.append(domainUid).append("-" + jobName + " -n ").append(namespace);
+    if (checkPodContains(cmdDelJob.toString(), jobName, jobName)) {
+      cmdDelJob = new StringBuffer("kubectl delete job ");
+      cmdDelJob.append(domainUid).append("-" + jobName + " -n ").append(namespace);
+      LoggerHelper.getLocal().log(Level.INFO, "Deleting job " + cmdDelJob);
+      exec(cmdDelJob.toString());
+    }
+
+    StringBuffer command = new StringBuffer();
+    command.append("kubectl create -f  ").append(scriptDir).append("/delete-domain-job.yaml");
+    ExecResult result = TestUtils.exec(command.toString());
+    if (!result.stdout().contains("sample-domain-job created")) {
+      throw new RuntimeException("FAILURE: delete domain job fails, exiting!");
+    }
+    StringBuffer cmd = new StringBuffer();
+    cmd.append("kubectl get pods ").append(" -n ").append(namespace).append(" | grep delete");
+
+    // check for completion
+    checkAnyCmdInLoop(cmd.toString(), "Completed");
+    command = new StringBuffer();
+    command.append("kubectl delete -f  ").append(scriptDir).append("/delete-domain-job.yaml");
+    TestUtils.exec(command.toString());
+  }
+
+  /**
+   * Run script to delete all DomainHome dirs in oke.
+   *
+   * @throws Exception exception
```

Comment:
no

Reply:
ok, at this point i started to get the feeling that this code is not ready to review, please fix and resubmit